test/functional/fundrawtransaction.py

```diff
@@ -452,7 +452,7 @@ def run_test(self):
         self.stop_node(2)
         self.stop_node(3)
         self.nodes[1].encryptwallet("test")
-        bitcoind_processes[1].wait(timeout=BITCOIND_PROC_WAIT_TIMEOUT)
+        self.bitcoind_processes[1].wait(60)
```

Comment:
Because I've moved the `BITCOIND_PROC_WAIT_TIMEOUT` constant into test_framework.py. The next PR (introduce TestNode) will remove `bitcoind_processes` dict entirely, so for this intermediate PR rather than add an import which I'll revert in the next PR, I've just changed this to a hard-coded value.